apps/design/frontend/src/geography_screen.tsx

```diff
@@ -406,10 +406,10 @@ function PrecinctForm({
   precinctId?: PrecinctId;
   savedPrecincts: Precinct[];
   districts: readonly District[];
-}): JSX.Element {
-  const [precinct, setPrecinct] = useState<Precinct>(
+}): JSX.Element | null {
+  const [precinct, setPrecinct] = useState<Precinct | undefined>(
```

Comment:
You can just infer the type parameter:

```suggestion
  const [precinct, setPrecinct] = useState(
```

Reply:
Huh, I thought I changed it in response to an error but it seems to work now 🤷